docs/source/getting-started/installation.rst

```diff
@@ -5,7 +5,7 @@ Installation
 
 .. note::
 
-    For Macs, macOS >= 10.15 is required to install SkyPilot. Apple Silicon-based devices (e.g. Apple M1) must run :code:`pip uninstall grpcio; conda install -c conda-forge grpcio=1.43.0` prior to installing SkyPilot.
+    For Macs, macOS >= 10.15 is required to install SkyPilot. Apple Silicon-based devices (e.g. Apple M1) must use :code:`python>=3.8` and run :code:`pip uninstall grpcio; conda install -c conda-forge grpcio=1.43.0` prior to installing SkyPilot.
```

Comment:
Hmm on my M1 Mac I've been using 3.7 until recently with no problems?

Reply:
Oh, interesting! Let me revert it then. Thanks for confirming!